drivers/mdio/Kconfig

```diff
@@ -1,49 +0,0 @@
-# MDIO configuration options
```

Comment:
Removing this file doesn't seem intentional?

Reply:
Yes, I will restore the original file.